app/scripts/constants/sentry-state.ts

```diff
@@ -271,6 +271,7 @@ export const SENTRY_BACKGROUND_STATE = {
     useTransactionSimulations: true,
     enableMV3TimestampSave: true,
   },
+  FeatureFlagController: {},
```

Comment:
Presumably we want feature flags in Sentry debug state, so we should add the feature flag property here

Reply:
Soon we plan on using the `anonymous` controller property to control this instead. I've put up a PR to adjust that there as well: https://github.com/MetaMask/core/pull/5004

Reply:
good idea ! added b5817983f2e305d0bffa8ea9cf577c490fa5bf21